utils/v2/event.ts

```diff
@@ -0,0 +1,52 @@
+import { api, Extrinsic } from "./setup";
+import { testLog } from "../Logger";
+import { User } from "../User";
+import { MangataGenericEvent, signTx } from "@mangata-finance/sdk";
+import { BN } from "@polkadot/util";
+import { getEventResultFromMangataTx } from "../txHandler";
+import { ExtrinsicResult } from "../eventListeners";
+
+// @ts-ignore
+export const logEvent = (phase, data, method, section) => {
+  testLog
+    .getLog()
+    .info(
+      phase.toString() + " : " + section + "." + method + " " + data.toString()
+    );
+};
+
+export const signSendFinalized = async (
```

Comment:
I think we have something like "signSendAndWaitForTxtofinish" but not 100% sure :laughing:

Reply:
it is mostly for logging and to have finer control what happens

the SDK signSend was not enough for me